packages/frontend/src/pages/explore/comics.tsx

```diff
@@ -24,12 +27,27 @@ const ComicSearch: NextPageWithLayout = () => {
     { enabled: !!value.length, initialData: [] },
   );
 
-  const handleAddComic = (comic: TComicItem) => {
-    addComic({
-      comicId: comic.id,
-      title: comic.title,
-      imageUrl: createImageUrl(comic.images),
-    });
+  const handleAddComic = async (comic: TComicItem) => {
```

Comment:
Have you considered using the `useMutation` hook by **react-query** for this?

It basically does what your doing, but in a nice way for you to easily consume and adjust when needed.

Reply:
Good point. I'll take a look!